src/components/vm/consoles/vnc.jsx

```diff
@@ -205,12 +218,29 @@ const VncEditModal = ({ vm, inactive_vnc }) => {
     );
 };
 
-export const VncActiveActions = ({ state, vm, vnc, onAddErrorNotification }) => {
+export const VncActiveActions = ({ state, vm, vnc, isExpanded, onAddErrorNotification }) => {
     const [isOpen, setIsOpen] = useState(false);
 
     if (!state.connected)
         return null;
 
+    let scale_resize_dropdown = null;
+    if (isExpanded) {
+        scale_resize_dropdown = (
+            <SimpleSelect
+                options={
+                    [
+                        { value: "none", content: _("No scaling or resizing") },
+                        { value: "local", content: _("Local scaling") },
+                        { value: "remote", content: _("Remote resizing") },
+                    ]
+                }
+                selected={state.sizeMode}
+                onSelect={val => state.setSizeMode(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2015-3fe0b058-20250716-111222-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)